src/main/java/com/mate/controller/driver/DeleteDriverController.java

```diff
@@ -0,0 +1,21 @@
+package com.mate.controller.driver;
+
+import com.mate.lib.Injector;
+import com.mate.service.DriverService;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.mate");
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
```suggestion
    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```
in all DeleteControllers make this method `doGet()`